src/utils/helper-functions/guards.ts

```diff
@@ -30,6 +31,5 @@ export const isObjectExpression = isNodeOfType(AST_NODE_TYPES.ObjectExpression)
 export const isProperty = isNodeOfType(AST_NODE_TYPES.Property)
 
 export function isTypeReference(type: ts.Type): type is ts.TypeReference {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  return Boolean((type as any).target)
+  return type.hasOwnProperty('target')
```

Comment:
Taking into consideration the fact that I was messing up this file, I changed the implementation here so we can remove the typecast.

Reply:
Good catch!